modules/decredprice.js

```diff
@@ -40,7 +40,20 @@ exports.module = function() {
                     callback("ERR");
                 }
             });
-        }, /*
+        },
+        USD: function(callback) {
+	    request('https://api.coinmarketcap.com/v1/ticker/decred/', function (error, response, body) {
+                if (!error && response.statusCode == 200) {
+                    var s = JSON.parse(body);
+                    var price = s[0].price_usd;
```

Comment:
This should round the price properly (untested):

var price = Math.round(s[0].price_usd*100)/100;